eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalStatements.scala

```diff
@@ -222,6 +230,11 @@ object JournalStatements {
                 metadata        = metadata,
                 headers         = headers,
               )
+
+              JournalRecord(
+                event        = eventRecord,
+                metaRecordId = row.decode[Option[RecordId]]("meta_record_id"),
+              )
```

Comment:
why we need this extra wrapper and why not put `metaRecordId` as field in `EventRecord`?

Reply:
Great question, I spend some time thinking what should I do.

`EventRecord` are created in `ReplicateRecords` in module `replicator` and passed down into `ReplicatedCassandra` in `cassandra` module. On level of `ReplicateRecords` I cannot provide instance of `RecordId` because metadata not yet available, it will be queried deeper in `cassandra` module because its purely Cassandra part, designed to overcome its limitations. On `replicator` level internal storage schema does not matter and its constraints should not leak out as well.

I choose having additional abstraction layer to avoid exposing `RecordId` in `replicator` (as unrelated to Replicator) and to avoid having case class with field that will be overrided later deeper in code.